Adds kickstart to the cli docs (#4238)
FusionAuth/fusionauth-site: astro/src/content/docs/customize/cli.mdx

* generate fake user data suitable for the [User Import API](/docs/apis/users#import-users)
* check an instance to see that [suggested common configuration](/docs/get-started/download-and-install/setup) has been performed

> * run demo instances of FusionAuth

Reviewer: ```suggestion
* run example instances of FusionAuth
```

  - `fusionauth email:upload` - Upload a specific template or all email templates to a FusionAuth server.
  - `fusionauth email:watch` - Watch the email template directory and upload changes to a FusionAuth server.
  - `fusionauth email:create` - Create a new email template locally.
- Fake user generation
  - `fusionauth import:generate` - Generate JSON for importing fake users for testing.
- Kickstart FusionAuth demos (beta)

>   - `fusionauth kickstart:install [dir]` - Generate the appropriate files to kickstart and install FusionAuth

Reviewer: I would suggest either rewording this to avoid using the generic term 'kickstart' or changing this to the proper noun Kickstart

```suggestion
  - `fusionauth kickstart:install [dir]` - Generate the appropriate files to Kickstart and install FusionAuth
```

>   - `fusionauth kickstart:install [dir]` - Generate the appropriate files to kickstart and install FusionAuth

Reviewer: ```suggestion
  - `fusionauth kickstart:install [dir]` - Generate the appropriate files to generate an example FusionAuth instance via Docker
```
an alternative

  - `fusionauth kickstart:start` - Run in the directory of a FusionAuth Docker image to run the image
  - `fusionauth kickstart:stop` - Run in the directory of a FusionAuth Docker image to stop the image

>   - `fusionauth kickstart:kill` - Run in the directory of a FusionAuth Docker image to shutdown and wipe the FusionAuth instance

Reviewer: ```suggestion
  - `fusionauth kickstart:start` - runs a FusionAuth Docker image, when executed in the same directory as the Docker configuration
  - `fusionauth kickstart:stop` - stops a FusionAuth Docker image, when executed in the same directory as the Docker configuration
```
IMO it's better to give the action before the qualifiers in cases like this. ditto for the following line (naturally, github is being buggy and won't let me make a suggestion on that line)

  - `fusionauth email:upload` - Upload a specific template or all email templates to a FusionAuth server.
  - `fusionauth email:watch` - Watch the email template directory and upload changes to a FusionAuth server.
  - `fusionauth email:create` - Create a new email template locally.
- Fake user generation
  - `fusionauth import:generate` - Generate JSON for importing fake users for testing.

> - Kickstart FusionAuth demos (beta)

Reviewer: ```suggestion
- Kickstart FusionAuth example instances (beta)
```
